### DISCUSSION TITLE
How to pass dynamic data to webhook request URL

### DISCUSSION THREAD

**Discussion by DeanKamali** [directus/directus#13725] [Archive]
Hey everyone, 

I'm playing around with Flows, and I'm liking it so far. 

I'm wondering if I were to lookup data from a collection, for example a URL is it possible to dynamically pass it to URL field web-hook request URL options menu? 

![image](https://user-images.githubusercontent.com/1252959/171968232-eaed070a-ab3d-405e-8d29-3884fa7b0918.png)
  
 Thank you

**Comment by rijkvanzanten** [comment]
Every option in operations can rely on the `{{}}` syntax to pull in a datapoint from another operation, you could configure the URL to be `https://example.com/{{ $trigger.payload.id }}` for example. To know what data you have available in your flow, check out the logs in the right sidebar from a previous run 🙂 If you want to lookup data from another collection for use in this URL, you will probably have to read that other data into the flow first using the Read Item operation in a previous step
